lib/engine-jit/src/builder.rs

```diff
@@ -4,16 +4,19 @@ use wasmer_compiler::{CompilerConfig, Features, Target};
 /// The JIT builder
 pub struct JIT {
     #[allow(dead_code)]
-    compiler_config: Option<Box<dyn CompilerConfig>>,
+    compiler_config: Option<Box<dyn CompilerConfig + Send + Sync>>,
```

Comment:
It's possible to make `Send` and `Sync` a requirement of the `CompilerConfig` itself so we don't have to write it everywhere.

Depends on if we ever think there will be a need for a `CompilerConfig` that can't be made thread safe

Reply:
Done! Thanks